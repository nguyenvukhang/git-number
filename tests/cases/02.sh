# staged file (filename)

init 0
touch gold
gitnu add gold # use filename
log gitnu status
