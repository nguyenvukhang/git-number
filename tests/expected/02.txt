On branch main

No commits yet

Changes to be committed:
1	[32mnew file:   gold[m

